sequences/couper_2.hcl

```diff
@@ -0,0 +1,74 @@
+server "client" {
+  hosts = ["*:8080"]
+  api {
+    endpoint "/" {
+      request "token" {
+        url = "http://localhost:8081/token"
+        form_body = {
+          sub = "myself"
+        }
+        expected_status = [200]
+      }
+      # The reference to backend_responses.token makes Couper wait for request "token"'s response.
+      request "pr" {
+        url = "http://localhost:8082/protected-res"
+        headers = {
+          authorization = "Bearer ${backend_responses.token.json_body.access_token}"
+        }
+        json_body = { a = true, b = 2 }
+      }
+#      request "ur" {
+#        url = "http://localhost:8082/unprotected-res"
+#        json_body = { c = "foo" }
+#      }
+      response {
+        json_body = {
+          pr = backend_responses.pr.json_body
+#          ur = backend_responses.ur.json_body
+        }
+      }
+    }
+  }
+}
+
+server "token-provider" {
```

Comment:
maybe some comments, that these servers are for demonstration purposes to have something to work on?